### ISSUE TITLE
Simplified presentation of Nullable/Option/Optional types

### ISSUE THREAD

**Issue by Crack** [TouK/nussknacker#770]
Currently these types are displayed on GUI as: plain type (eg. Long, the same as real Long type), Option, Optional. 

User shouldn't care about this - things should be optional or not. If possible we should hide Option/Optional from user interface. We could display these types as "{originalType}?", eg. "Long?". That notation could be also used in autocompletion popups to indicate nullability of parameters and returned values.

**Comment by lciolecki** [comment]
This don't need design. It's development case to show `Long` instead of `Option[Long]` / `Long` instead of `Optional[Long]`. Am I right @Crack?

**Comment by witekw** [comment]
Do we want to show that a field is mandatory or optional? Do we show "This field is mandatory and can not be empty" just after opening the modal when there is no default value?

**Comment by fi-** [comment]
After opening node details user should be able to identify required fields. We need to visually indicate which fields are required ones.
"This field is mandatory and can not be empty" - this red message and red borders around input should not be visible just after opening node. It should appear after validation. We should not validate before user fills the input.

**Comment by arkadius** [comment]
@lciolecki IMO we shouldn't handle Option/Optional, Nullable is enough. @fi- has right that we should switch from "punishing logic" because user not filled something to "advising logic", what should be filled.

**Comment by lciolecki** [comment]
I agree with that we should run validation after put something in input or after press save button.
